vhdbuilder/packer/windows-image.env

```diff
@@ -4,18 +4,18 @@
 # CLI example to get the latest image version:
 #   az vm image show --urn MicrosoftWindowsServer:WindowsServer:2019-Datacenter-Core-smalldisk:latest
 WINDOWS_2019_BASE_IMAGE_SKU=2019-Datacenter-Core-smalldisk
-WINDOWS_2019_BASE_IMAGE_VERSION=17763.4737.230802
+WINDOWS_2019_BASE_IMAGE_VERSION=17763.4851.230905
```

Comment:
could you help to add links in pr description for these kbs?

Reply:
Thanks for the reminder. Added it.